pandas/core/indexes/api.py

```diff
@@ -233,6 +233,7 @@ def _unique_indices(inds, dtype) -> Index:
         Parameters
         ----------
         inds : list of Index or list objects
+        dtype : dtype to set for the resulting Index
```

Comment:
above, it still says `The final dtype is inferred`.

Is this still true? It is a positional arg and not an optional kwarg, but we can explicitly pass `None` to get inference?

Reply:
Only if an all list arg is passed, so yeah should update docstring to reflect this